### PR TITLE
29611/accessible modal cypress

### PR DESCRIPTION
**Pull request by CBonade**
## Description
This converts the Accessible Modal test to Cypress to start our journey on deprecating Nightwatch in content-build.

## Testing done
Test passed in all local runs

### CODE REVIEW

File: src/platform/site-wide/tests/accessible-modal.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const lastModalItem = 'a[href="https://www.veteranscrisisline.net/"]';
+
+describe('Accessible Modal Test', () => {
+  it('Modal behaves appropriately in line with key presses', () => {
+    cy.visit('/');
+
+    // Open modal
+    cy.get(firstOpenControl)
+      .focus()
+      .realPress('Enter');
```
Comment (Rleahy22) [inline src/platform/site-wide/tests/accessible-modal.cypress.spec.js:16]: This is nice 😎

File: src/platform/testing/e2e/cypress/support/commands/keyboard.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Cypress.Commands.add('keys', keys => {
+  keys.forEach(key => {
+    cy.realPress(key);
+  });
+});
+
+Cypress.Commands.add('repeatKey', (key, multiple) => {
```
Comment (Rleahy22) [inline src/platform/testing/e2e/cypress/support/commands/keyboard.js:7]: 🚀

**Review by Rleahy22 — APPROVED** [review]

**Review by holdenhinkle — APPROVED** [review]
Nice - LGTM!

File: package.json [hunk 3]
```diff
@@ -205,6 +205,7 @@
   "dependencies": {
     "blob-polyfill": "^4.0.20200601",
     "core-js": "^3.17.3",
+    "cypress-real-events": "^1.5.1",
```
Comment (ddzz) [inline package.json:208]: This should be moved to `devDependencies`.

Comment (ddzz) [inline package.json:208]: This should be moved to `devDependencies`.

**Review by ddzz — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (CBonade) [reply package.json]: Went ahead and fixed this!

**Review by ddzz — APPROVED** [review]

**Review by theodur — APPROVED** [review]

**Pull request merged**